qiskit/algorithms/optimizers/scipy_optimizer.py

```diff
@@ -85,6 +85,16 @@ def get_support_level(self):
             "initial_point": self._initial_point_support_level,
         }
 
+    @property
+    def settings(self) -> Dict[str, Any]:
+        settings = {
+            "name": self._method,
```

Comment:
I think this should be `"method"` ?

Reply:
Now that we don't carry the `name` along anymore yeah, this should be `method`